extensions/amp-date-picker/validator-amp-date-picker.protoascii

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-date-picker"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {
+  html_format: AMP
+  spec_name: "amp-date-picker[type=single][mode=static]"
+  tag_name: "AMP-DATE-PICKER"
+  requires_extension: "amp-date-picker"
+  attr_lists: "amp-date-picker-common-attributes"
+  attr_lists: "amp-date-picker-single-type-attributes"
+  attr_lists: "amp-date-picker-static-mode-attributes"
+  attr_lists: "extended-amp-global"
+  attrs: {
+    name: "mode"
+    value_regex_casei: "static|"
```

Comment:
Just to confirm, you want this to also match an empty string? That is this would test for either:

Reply:
My goal here was to have it be valid if the attribute is not present.

Reply:
Then you don't need it. Only way to force an attribute to be present is with `mandatory: true`